arc/src/main.c

```diff
@@ -762,55 +768,132 @@ static int stop_gyro_trigger(struct device *dev)
     return 0;
 }
 
+#ifdef BUILD_MODULE_SENSOR_LIGHT
+static double cube_root_recursive(double num, double low, double high) {
```

Comment:
the w3c api expects an ambient light value returned from the callback, now if the zephyr driver supports the grove light, then the driver could convert the value, but this grove light is not supported